app/views/_feedback_on_appeals_to_crown_court.html.haml

```diff
@@ -0,0 +1,10 @@
+.govuk-notification-banner{ role: "region", aria: { labelledby: "govuk-notification-banner-title" }, data: { module: "govuk-notification-banner" } }
+  .govuk-notification-banner__header
+    %h2.govuk-notification-banner__title#govuk-notification-banner-title
+      Important
+  .govuk-notification-banner__content
+    %p.govuk-notification-banner__heading
+      Give
```

Comment:
It would be really nice if we could be consistent about keeping copy in the i18n yml files (I agree that since this will never be translated, we don't really need them, but it's a pain when almost all the copy is there and then some is hiding in the view files

Reply:
I see your point, but
- It’s temporary.
- Only one language is supported.
- Adding it to en.yml would just add noise you’ll delete later.